### ISSUE TITLE
[Bug] [5.3.0] Cannot find to-jyutping module in build from source code

### ISSUE THREAD

**Issue by joshuactk** [hockyy/miteiru#62]
**Describe the bug**
I compiled and installed Miteiru 5.3.0 from source code for my Linux machine. It would not run because of to-jyutping.

The output in my terminal shows:
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/opt/miteiru/resources/app.asar/node_modules/to-jyutping/dist/index.cjs'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1269:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1257:15)
    at resolveExports (node:internal/modules/cjs/loader:641:14)
    at Module._findPath (node:internal/modules/cjs/loader:731:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1218:27)
    at s._resolveFilename (node:electron/js2c/browser_init:2:121498)
    at Module._load (node:internal/modules/cjs/loader:1058:27)
    at c._load (node:electron/js2c/node_init:2:17025)
    at Module.require (node:internal/modules/cjs/loader:1318:19)
    at require (node:internal/modules/helpers:179:18)

**To Reproduce**
Upon extracting the source code and entering the folder through the terminal,
1. npm install
2. npm run script:initrepo
3. npm run build:linux22
4. cd dist
5. sudo dpkg -i miteiru-5.3.0-linux-22.04-amd64.deb

**Expected behavior**
I expected the application to open, but it could not.

**Screenshots**
![Screenshot_20240926_165429](https://github.com/user-attachments/assets/a0ce44e8-2db2-40b8-8185-b9dd610ab811)

**Desktop:**
 - OS: Kubuntu 
 - Version 24.04 

**Additional context**

**Comment by hockyy** [comment]
Can you check whether it builds in the new version 5.5.0 @joshuactk

**Comment by hockyy** [comment]
https://github.com/hockyy/miteiru/releases/tag/v5.5.0

**Comment by hockyy** [comment]
The binary compiled from my pc runs in mine

**Comment by joshuactk** [comment]
I got to build v5.5.0 in Linux, just needed to specify the homepage in package.json

When running the program built from source code, I got 2 errors:
![Screenshot_20241003_230636](https://github.com/user-attachments/assets/76e5e5d3-2b7b-4b87-aa71-e1f79537efd3)
- The first one from starting miteiru
- The second one from clicking the "un, chotto miteiru dake" button
![Screenshot_20241003_231142](https://github.com/user-attachments/assets/52c2cce1-a62d-46c9-a3b3-05ea47611b61)
- First error with Kuromoji
- Second error with Mecab
- Third error with Jieba (Chinese), which freezes the program
![Screenshot_20241003_231306](https://github.com/user-attachments/assets/473867f3-f3b1-442f-b83d-d1d7be7ae18b)
- With Jieba (Cantonese)

I'll try what happens with the .deb you built in the Releases section and let you know.

Let me also know if you need me to elaborate on the dependency versions I have on my system or whatnot.

**Comment by joshuactk** [comment]
![Screenshot_20241003_232521](https://github.com/user-attachments/assets/a8ad0f88-9670-4ce2-b4f4-59e884ba2b48)
Your build works like a dream

**Comment by hockyy** [comment]
@joshuactk maybe it has something to do with build tools lack of support or something, 🤔🤔🤔 I'm not sure

**Comment by hockyy** [comment]
https://github.com/hockyy/miteiru/releases/tag/assets

@joshuactk you forget to download this and put it into miteiru/archived/public.zip, then run `npm run script:initrepo`

**Comment by joshuactk** [comment]
The Linux build works perfectly after I extracted the contents of the zip directly into the `miteiru/archived/`. It did not work when I extracted it initially as `miteiru/archived/public/`, but that's fine.

Thank you so much for the help! This program is a godsend.

I suggest including the direct extraction of the zip file into `miteiru/archived/` as a step in the Own Build Installation Guide of the `README.md` for less informed folks who might run into a similar issue.

I'll also try building this in my gf's Mac laptop over the weekend with the above step in mind.
